tools/cli/pkg/command/options.go

```diff
@@ -137,13 +138,15 @@ func (keys *GlobalOptionsKey) GardenerNamespace() string {
 
 // SetOutputOpt configures the optput type option on the given command
 func SetOutputOpt(cmd *cobra.Command, opt *string) {
-	cmd.Flags().StringVarP(opt, "output", "o", tableOutput, fmt.Sprintf("Output type of displayed Runtime(s). The possible values are: %s, %s.", tableOutput, jsonOutput))
+	cmd.Flags().StringVarP(opt, "output", "o", tableOutput, fmt.Sprintf("Output type of displayed Runtime(s). The possible values are: %s, %s, %s.", tableOutput, jsonOutput, customcolumnOutput))
```

Comment:
I think we should add a small description for accepted format of `custom-columns=...`

Reply:
Now add the description in the `kcp orchestration/runtimes` 's corbra definition.
e.g. 
`kcp orchestration -o custom="Orchestration ID:{.OrchestrationID},STATE:{.State},CREATED AT:{.createdAt}"` 
`kcp runtimes -c bbc3ee7 -o custom="INSTANCE ID:instanceID,SHOOTNAME:shootName"`
`kcp runtimes -o custom="INSTANCE ID:instanceID,SHOOTNAME:shootName,runtimeID:runtimeID,STATUS:{status.provisioning}"`